Error running all library tests

Built as follows:
```
$ ./build.cmd -subset clr -c Checked
$ ./build.cmd -subset Libs+Libs.Tests -runtimeConfiguration Checked
```
Then I tried to run all tests from root as follows (per a comment by @ViktorHofer in Issue #34152):
```
$ ./build.cmd libs.tests -test -coverage -runtimeConfiguration Checked
```

This ended quickly in the following error:

> Build FAILED.
> 
> C:\Users\mrj00\source\repos\runtime\eng\testing\coverage.targets(42,7): error MSB4184: The expression ""C:\Users\mrj00\source\repos\runtime\src\libraries".SubString(50)" cannot be evaluated. startIndex cannot be larger than length of string. (Parameter 'startIndex') [C:\Users\mrj00\source\repos\runtime\Build.proj]
>     0 Warning(s)
>     1 Error(s)
> 
> Time Elapsed 00:00:23.25
> Build failed.
> Some builds failed:
>         Configuration: Debug, Architecture: x64
>

@ViktorHofer 
I got 256 errors all similar to this:

> C:\Users\mrj00\source\repos\runtime\eng\testing\tests.targets(102,24): error MSB4113: Specified condition "'$(TargetsMobile)'" evaluates to "" instead of a boolean. [C:\Users\mrj00\source\repos\runtime\src\libraries\Common\tests\Common.Tests.csproj]

Is there an expected date for that PR to be applied?

--- a/eng/testing/tests.targets
+++ b/eng/testing/tests.targets
@@ -99,7 +99,7 @@
     <PropertyGroup>
       <RunTestsCommand>"$(RunScriptOutputPath)" --runtime-path "$(TestHostRootPath.TrimEnd('\/'))"</RunTestsCommand>
       <RunTestsCommand Condition="'$(TestRspFile)' != '' and '$(RuntimeFlavor)' != 'Mono'">$(RunTestsCommand) --rsp-file "$(TestRspFile)"</RunTestsCommand>
-      <RunTestsCommand Condition="'$(TargetsMobile)'">"$(RunScriptOutputPath)" $(AssemblyName) $(TargetArchitecture)</RunTestsCommand>
+      <RunTestsCommand Condition="'$(TargetsMobile)' == 'true'">"$(RunScriptOutputPath)" $(AssemblyName) $(TargetArchitecture)</RunTestsCommand>
     </PropertyGroup>
 
     <!-- Invoke the run script with the test host as the runtime path. -->

